manifests/charts/gateways/istio-egress/templates/_affinity.tpl

```diff
@@ -2,32 +2,10 @@
 
 {{ define "nodeaffinity" }}
 nodeAffinity:
-  requiredDuringSchedulingIgnoredDuringExecution:
-  {{- include "nodeAffinityRequiredDuringScheduling" . }}
   preferredDuringSchedulingIgnoredDuringExecution:
   {{- include "nodeAffinityPreferredDuringScheduling" . }}
 {{- end }}
 
-{{- define "nodeAffinityRequiredDuringScheduling" }}
-      nodeSelectorTerms:
-      - matchExpressions:
-        - key: kubernetes.io/arch
-          operator: In
-          values:
-        {{- range $key, $val := .global.arch }}
```

Comment:
I think we still need ability to set it, just not default?

Reply:
I added a value to decide whether we want to set this. PTAL.